server/routes/activities/record-attendance/recordAttendanceRequests.ts

```diff
@@ -3,6 +3,13 @@ export type SessionCancellationRequest = {
   comment?: string
 }
 
+export enum AttendActivityMode {
+  MULTIPLE = 'MULTIPLE',
+  SINGLE = 'SINGLE',
+}
+
 export type RecordAttendanceRequests = {
-  sessionCancellation: SessionCancellationRequest
+  sessionCancellation?: SessionCancellationRequest
+  mode?: AttendActivityMode
+  selectedInstanceIds?: string[]
```

Comment:
The scheduled instances that the user selected which can be used further down the journey and when the ui navigates back.